test/styles/RRR.R

```diff
@@ -39,38 +48,110 @@ fun_call(argument1,
 fun_call(
     argument1,
     argument2
-    )
+)
 
 ## 3
 fun_call(parameter = (
     stuff
-    ),
-         argument)
+),
+argument)
 
 ## 4
 fun_call(parameter = fun_argument(
-             stuff
-    ),
-         argument)
+    argument1
+),
+argument2)
+
+## 5
+fun_call(parameter = fun_argument(argument1,
+                                  argument2
+                                  )
+        ,
+         argument3)
```

Comment:
Note that indentation after `{` is controlled by `ess-offset-block` and is independent of all this. `ess-indent-from-outer-parameter` also has an effect for blocks, which I forgot to mention in the doc.